nessie/externals/redshift.py

```diff
@@ -96,12 +96,12 @@ def copy_tsv_from_s3(table, s3_key):
 
 def create_external_schema(external_schema, role):
     query = """
-        CREATE EXTERNAL SCHEMA {external_schema}
```

Comment:
Is there a particular reason to avoid our usual `f` quote here? Seems like it would be more readable.

Reply:
Switch to using f strings @raydavis.